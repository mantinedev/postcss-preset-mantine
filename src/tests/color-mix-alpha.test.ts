import { testTransform } from './utils';

const input = `
.demo {
  background: alpha(#f00, 0.5);
  border: rem(1px) solid alpha(var(--mantine-color-gray-4), 0.1);
}
`;

describe('color-mix-alpha', () => {
  it('correctly mixes colors', async () => {
    const res = await testTransform(input);
    expect(res.css).toMatchSnapshot();
  });
});
