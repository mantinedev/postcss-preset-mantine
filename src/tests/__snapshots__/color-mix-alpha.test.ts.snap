// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`color-mix-alpha correctly mixes colors 1`] = `
"
.demo {
  background: color-mix(in srgb, #f00, transparent 50%);
  border: calc(0.0625rem * var(--mantine-scale)) solid color-mix(in srgb, var(--mantine-color-gray-4), transparent 90%);
}
"
`;
